engines/mads/mads.cpp

```diff
@@ -77,6 +78,12 @@ MADSEngine::~MADSEngine() {
 }
 
 void MADSEngine::initialize() {
+	if (scumm_stricmp(_gameDescription->desc.filesDescriptions[0].fileName, "mpslabs.001") == 0) {
```

Comment:
It would be cleaner to add a game flag for this in the detection tables, e.g. GF_COMPRESSED, instead of checking for data file names

Reply:
Done